pkg/stanza/fileconsumer/metadata.yaml

```
@@ -9,8 +9,31 @@ status:
     active: [andrzej-stencel]
     emeritus: [djaglowski]
 
+attributes:
+  file_path:
+    description: The path of the file being monitored
+    type: string
+
 telemetry:
   metrics:
+    fileconsumer_file_offset_bytes:
```

I think unit is not supposed to be part of the metric name https://opentelemetry.io/docs/specs/semconv/general/metrics/#units.
```suggestion
    fileconsumer_file_offset:
```